src/app/cube/about-us/components/about-clark/about-clark.component.html

```diff
@@ -3,21 +3,40 @@
     
     <div class="first-section">
       <div class="fs-text-container">
-        <p>CLARK is a platform for <span class="blue_text" >BUILDING</span> and sharing free cybersecurity curriculum. It includes three components – the model for building curriculum, the digital library system, and the curriculum collections. It provides cybersecurity faculty the building blocks to train the <span class="blue_text">NEXT WAVE</span> of researchers and better prepare the cybersecurity workforce.</p>
+        <p>
+          CLARK hosts the largest compilation of free, high-value, high-impact <span class="blue_text">CYBERSECURITY</span> 
+          curriculum created by top researchers that has been peer-reviewed by accredited instructional designers and 
+          subject matter experts. Instructors looking to teach new cybersecurity <span class="blue_text" >COURSES</span> or 
+          expand and improve their own instruction, can find ready-to-use modular content including videos, 
+          labs, and slides that can be reused and modified under Creative Commons 4.0.
+        </p>
       </div>
       <img class = "fs-img" src="../../../../../assets/images/collections/secinj.png" alt="Code Responsibly">
     </div>
       
     <div class="second-section">
       <img class = "ss-img" src="../../../../../assets/images/module.JPG" alt="Module Example">
       <div class="ss-text-container">
-        <p>CLARK hosts the largest compilation of high-value, high-impact <span class="blue_text">CYBERSECURITY</span> curriculum created by top researchers, peer-reviewed by instructional designers and subject matter experts. Instructors looking to teach new cybersecurity <span class="blue_text" >COURSES</span> or expand and improve their own instruction, can find ready-to-use modular content, including videos, labs, and slides that can be modified under the Creative Commons license.</p>
+        <p>
+          CLARK supports the model for <span class="blue_text" >BUILDING</span>  
+          modularized curriculum, the digital library system to <span class="blue_text" >BROWSE</span> 
+          free curriculum, and unique <span class="blue_text" >COLLECTIONS</span>
+          to categorize distinct groups of curriculum. CLARK provides cybersecurity educators the tools needed
```

Comment:
Preferably on line 24 it would be better to reword the sentence "CLARK provides cybersecurity educators the tools needed..." as "CLARK provides cybersecurity educators _with_ the tools needed..."